geometry_msgs/Vector3 position
float32 time
float32 acc
---
bool success